docsrc/markdown/datafeed.md

```diff
@@ -155,6 +155,109 @@ datafeed:
     maxIntervalMillis: 300000
 ```
 
+# Datahose
+Datahose is very similar to datafeed: it enables a bot to receive [_Real Time
+Events_](https://docs.developers.symphony.com/building-bots-on-symphony/datafeed/real-time-events) with the main
+difference that *all* events of the pod are received. The datahose loop is a core service built on top of the events API
+and provide a dedicated contract to bot developers to work with datahose. This is compatible with agent version 20.15 onwards.
```

Comment:
```suggestion
and provide a dedicated contract to bot developers to work with Datahose. This is compatible with Agent version 20.15 onwards.
```

in the doc (like https://docs.developers.symphony.com/building-bots-on-symphony/datafeed), Datafeed, Agent, and so are capitalized